packages/react-scripts/template/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "React App",
+  "icons": [
+    {
+      "src": "favicon.ico",
+      "sizes": "192x192",
+      "type": "image/png"
```

Comment:
We should probably add documentation explaining to update these fields.
Are the `type` and `sizes` fields necessary? We want to avoid having to update multiple locations to get things working -- if we could automatically deduce this based on `index.html` that'd be even better (autogen this entire file at build instead of having it premade).